components/util/opts.rs

```diff
@@ -767,7 +768,14 @@ pub fn from_cmdline_args(args: &[String]) -> ArgumentParsingResult {
     // This must happen after setting the default options, since the prefs rely on
     // on the resource path.
     for pref in opt_match.opt_strs("pref").iter() {
-        prefs::set_pref(pref, PrefValue::Boolean(true));
+        let split: Vec<&str> = pref.split('=').collect();
```

Comment:
@Manishearth well, that's the old diff to which you added the comment :) Here is the updated one: https://github.com/servo/servo/pull/10204/files#diff-0543c2dfa73f1f316c2846ebb815deb5R771